packages/common/src/FormattingUtils.js

```diff
@@ -3,6 +3,8 @@ const networkUtils = require("./PublicNetworks");
 const BigNumber = require("bignumber.js");
 const moment = require("moment");
 const { formatFixed, parseFixed } = require("@ethersproject/bignumber");
+const web3 = require("web3");
+const { toBN } = web3.utils;
```

Comment:
nit: it's idiomatic (when using web3) to name the _library_ `Web3` and the instance (result of `new Web3(provider)`) `web3`.

```suggestion
const Web3 = require("web3");
const { toBN } = Web3.utils;
```

Reply:
I chose to remove this in full and inject the `web3` object like we do with the other formatters in this file.